src/refmvs.rs

```diff
@@ -142,6 +150,9 @@ pub struct refmvs_block(pub refmvs_block_unaligned);
 
 #[repr(C)]
 pub(crate) struct refmvs_frame {
+    /// A pointer to a `struct refmvs_frame` may be passed to a function of type `load_tmvs_fn`.
+    /// However, the `frm_hdr` pointer is not accessed in such a function. Thus, it is
+    /// safe to have a pointer to `Rav1dFrameHeader` instead of `Dav1dFrameHeader` here
```

Comment:
```suggestion
    /// A pointer to a [`refmvs_frame`] may be passed to a [`load_tmvs_fn`] function.
    /// However, the [`Self::frm_hdr`] pointer is not accessed in such a function (see [`load_tmvs_c`]).
    /// Thus, it is safe to have a pointer to [`Rav1dFrameHeader`] instead of [`Dav1dFrameHeader`] here.
```

This is unfortunate, as previously `refmvs_frame` was not passed to asm and didn't need to remain `#[repr(C)]` and compatible.  Luckily this field isn't used in asm `fn`s, though now I'm worried that it may in the future.  Hopefully not, as that would mess up a lot of things.

Eventually, I think we should make this field `pub _frm_hdr: *const ()` and pass `&Rav1dFrameHeader` down the callstack where it's needed (if this possible, which I hope it is, since no `Rav1dRef` is stored here).  That fixes the lifetime issue of the pointer and makes it clear that this field should not be accessed.